Sources/Sentry/SentrySDK.m

```diff
@@ -42,24 +42,32 @@ + (void)setCurrentHub:(SentryHub *)hub {
 }
  
 + (id)initWithOptions:(NSDictionary<NSString *,id> *)optionsDict {
+    [SentrySDK startWithOptions:optionsDict];
```

Comment:
Can we mark the old init functions as deprecated?

Reply:
They are both marked as deprecated
https://github.com/getsentry/sentry-cocoa/pull/441/files#diff-e98d4373fbfc44d86abfa859f8883ee3R36-R45

Reply:
Ah sorry, didn't see it. Thanks for pointing it out.